app/views/projects/samples/attachments/_attachment.html.erb

```diff
@@ -1,19 +1,75 @@
 <tr
```

Comment:
**nitpick:** Can you check your formatters?  This file's indentations are off.  Are you using `erb-formatter`?